src/schema/rules/files/raw/eyetrack.yaml

```diff
@@ -0,0 +1,15 @@
+---
+eyetrack:
+  suffixes:
+    - eyetrack
+  extensions:
+    - .json
+    - .tsv.gz
+  datatypes:
+    - eyetrack
```

Comment:
Since eyetracking data doesn't go in its own folder, this should reflect all of the corresponding datatypes eyetracking can be acquired with. I think you can base this off of the `physio` group in `src/schema/rules/files/raw/task.yaml`.

Reply:
gone